cmd/skaffold/app/cmd/flags.go

```diff
@@ -277,7 +277,7 @@ var flagRegistry = []Flag{
 		Usage:         "Wait for deployed resources to stabilize",
 		Value:         &opts.StatusCheck,
 		DefValue:      true,
-		FlagAddMethod: "BoolVar",
+		FlagAddMethod: "Var",
```

Comment:
we should also add `NoOptDefVal:   "true"`. This lets us invoke `skaffold foo --status-check` without the `=true` argument.

Reply:
Done, thanks!